browser/extensions/api/brave_extensions_api_client.cc

```diff
@@ -25,6 +26,19 @@ bool BraveExtensionsAPIClient::ShouldHideBrowserNetworkRequest(
                         base::CompareCase::INSENSITIVE_ASCII))) {
     return true;  // protected URL
   }
+
+  URLPattern pattern1(URLPattern::SCHEME_HTTPS,
```

Comment:
It probably doesn't matter in practice, but the reason why I checked the domain instead of a URL pattern was to also cover any `http://` URLs. Not that we should be using any `http://` URLs anywhere...

Reply:
I don't think http can be used here

Reply:
You're right, I completely missed that for some reason.